zkvm/src/types.rs

```diff
@@ -169,13 +169,19 @@ impl Data {
             Data::Input(input) => input.encode(buf),
         };
     }
+
+    /// Returns an empty Data type.
+    pub fn empty() -> Self {
```

Comment:
`impl Default for Data` is a good choice for this because it can then be used with all the generic APIs that use `T: Default`.